<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Default test"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.054">
    <properties></properties>

            <testcase
                name="advocateScheduleEventTest"
                time="0.054"
                classname="cuesz.demo.iostest.Case02_Homescreen_spherespearatordemotest">

            <failure type="java.lang.NullPointerException" message="Cannot invoke &quot;io.appium.java_client.AppiumDriver.executeScript(String, Object[])&quot; because &quot;this.driver&quot; is null">
                <![CDATA[java.lang.NullPointerException: Cannot invoke "io.appium.java_client.AppiumDriver.executeScript(String, Object[])" because "this.driver" is null
                cuesz.utils.AppiumappUtils.launchAppWithPackageId(AppiumappUtils.java:22)cuesz.demo.ios.Case02_Homescreen_spherespearatordemo.Homescreen(Case02_Homescreen_spherespearatordemo.java:51)cuesz.demo.iostest.Case02_Homescreen_spherespearatordemotest.advocateScheduleEventTest(Case02_Homescreen_spherespearatordemotest.java:55)java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)java.base/java.lang.reflect.Method.invoke(Method.java:580)org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)org.testng.internal.Invoker.invokeMethod(Invoker.java:661)org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)org.testng.TestRunner.privateRun(TestRunner.java:744)org.testng.TestRunner.run(TestRunner.java:602)org.testng.SuiteRunner.runTest(SuiteRunner.java:380)org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)org.testng.SuiteRunner.run(SuiteRunner.java:289)org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)org.testng.TestNG.runSuitesLocally(TestNG.java:1226)org.testng.TestNG.runSuites(TestNG.java:1144)org.testng.TestNG.run(TestNG.java:1115)org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </failure>
    </testcase>


</testsuite>
